packages/network-of-terms-catalog/catalog/datasets/filmmuseum-genres.rq

```diff
@@ -0,0 +1,37 @@
+{
+  "@context": "https://schema.org/docs/jsonldcontext.jsonld",
+  "@id": "http://data.beeldengeluid.nl/gtaa/Genre_Filmmuseum",
+  "@type": "Dataset",
+  "name": [
+    {
+      "@language": "nl",
+      "@value": "Filmmuseum: Genres"
+    }
+  ],
+  "creator": [
+    {
+      "@id": "https://www.beeldengeluid.nl/"
+    }
+  ],
+  "url": [
+    "http://data.beeldengeluid.nl/gtaa/Genre_Filmmuseum"
+  ],
+  "distribution": [
+    {
+      "@id": "https://data.beeldengeluid.nl/id/datadownload/0033",

```

Comment:
Should these IRIs be resolvable? Currently they are not.

Reply:
yes, now they are. the data catalog needed to be updated.